src/plugin/analytics/Resources/modules/administration/dashboard/store/selectors.js

```diff
@@ -1,6 +1,7 @@
 import {createSelector} from 'reselect'
 
 const STORE_NAME = 'dashboard'
+const LIST_NAME = STORE_NAME + '_log_list'
```

Comment:
The list name is the path in the store object.
Regarding your reducer, this should be : `STORE_NAME+'.list'`.

Reply:
Change done, thx :)